### DISCUSSION TITLE
v0.8.0

### DISCUSSION THREAD

**Discussion by WilliamYue37** [TensorAuto/OpenTau#441] [Announcements]
## What's Changed
* fix(policies): respect skip_normalization_weights on checkpoint load by @shuheng-liu in https://github.com/TensorAuto/OpenTau/pull/425
* Add Cosmos3-Super 64B action-generation benchmark (policy mode) by @WilliamYue37 in https://github.com/TensorAuto/OpenTau/pull/426
* docs(models): document all policies on the models page by @shuheng-liu in https://github.com/TensorAuto/OpenTau/pull/427
* feat(train): optional torch.compile of flow-matching module for pi05/pi07 by @WilliamYue37 in https://github.com/TensorAuto/OpenTau/pull/428
* feat(deepspeed): validate ZeRO-3 for pi05 full fine-tune + ZeRO-2 benchmark by @WilliamYue37 in https://github.com/TensorAuto/OpenTau/pull/430
* feat(robocasa): per-task official horizon for sim eval by @shuheng-liu in https://github.com/TensorAuto/OpenTau/pull/431
* feat(eval): make eval simulation seeding reproducible and configurable by @shuheng-liu in https://github.com/TensorAuto/OpenTau/pull/433
* Adding learnable embeddings for modality specific by @akshay18iitg in https://github.com/TensorAuto/OpenTau/pull/432
* feat(policies): add cosmos3 (Qwen3-VL-32B reasoner + flow-matching expert) by @WilliamYue37 in https://github.com/TensorAuto/OpenTau/pull/429
* feat(policies): optional knowledge_insulation + skip zero-weight loss terms by @shuheng-liu in https://github.com/TensorAuto/OpenTau/pull/437
* feat(pi05_mem): RLDX-1 STSS motion module video encoder by @akshay18iitg in https://github.com/TensorAuto/OpenTau/pull/438
* feat(grpc): optional x-tuner-api-key authentication by @cenxiaoxinxin-dotcom in https://github.com/TensorAuto/OpenTau/pull/439
* docs(cosmos3): add tensor-flow ASCII diagram to model docs by @WilliamYue37 in https://github.com/TensorAuto/OpenTau/pull/434
* chore: bump version to v0.8.0 by @WilliamYue37 in https://github.com/TensorAuto/OpenTau/pull/440

## New Contributors
* @cenxiaoxinxin-dotcom made their first contribution in https://github.com/TensorAuto/OpenTau/pull/439

**Full Changelog**: https://github.com/TensorAuto/OpenTau/compare/v0.7.0...v0.8.0

<hr /><em>This discussion was created from the release <a href='https://github.com/TensorAuto/OpenTau/releases/tag/v0.8.0'>v0.8.0</a>.</em>
